docs/product-user-guide/src/main/asciidoc/con-business-central-settings-library.adoc

```diff
@@ -0,0 +1,34 @@
+[[con-business-central-settings-library]]
+= Library
+ 
+The settings in the Library perspective determine the behavior of the Projects perspective, which is available from the *Main* menu.
+
+The Library Preferences section enables you to specify a URL for the examples repository. Example projects will be fetched from this repository and will be displayed on Library Import Examples perspective. If you do not specify a URL, the default examples will be displayed. The Library Preferences section has the following subsections:
+COMMENT: How can I see the examples?
+
+.File Exporting
+
+This section enalbes you set the orientation, units, and page format for exporting PDF files. 
+
+.Organizational Units
+
+In this section, you can specify the following default values:
+
+* The default team name if no team exists
+* The default team owner if no team exists
+* The default team group ID 
+* A singular alias that will be used instead of team name
+* A plural alias that will be used instead of team name
+COMMENT: Please clarify the differences in the plural and singular aliases. When would I use one or the other?
```

Comment:
Make sure to remove the note. To answer the question:
![pluralvssingular](https://user-images.githubusercontent.com/2716069/31704649-8d00d2a2-b3e2-11e7-8812-09a0fdf27439.png)

A word of explanation about why we even have this setting:
In the past the "group of repositories / projects" used to be called "organizational unit", which people found confusing. Now it's called Teams. The way it's called in particular user's organization might differ, therefore it's been made configurable. I think in the near future it will be renamed to "spaces" by default :)